code/modules/transport/pods/secondary_system.dm

```diff
@@ -909,3 +909,72 @@
 		dispense()
 	in_bump = 0
 	return
+
+/obj/item/shipcomponent/secondary_system/syndicate_rewind_system
+	name = "Syndicate Rewind System"
+	desc = "An unfinished pod system, the blueprints for which have been plundered from a raid on a now-destroyed Syndicate base. Requires a unique power source to function."
+	power_used = 50
+	f_active = 1
+	hud_state = "SRS_icon"
+	var/cooldown = 0
+	var/core_inserted = false
+	var/health_snapshot
+	var/image/rewind
+	icon = 'icons/misc/retribution/SWORD_loot.dmi'
+	icon_state= "SRS_empty"
+
+	Use(mob/user as mob)
+		activate()
+		return
+
+	deactivate()
+		return
+
+	activate()
+		if(!core_inserted)
+			boutput(ship.pilot, "<span class='alert'><B>The system requires a unique power source to function!</B></span>")
+			return
+		else if(cooldown > world.time)
```

Comment:
Rather than using `world.time` which is a measure game time which is vulnerable to being slowed by overtime, consider using `TIME` which uses real world time where applicable

Reply:
Done.